.github/workflows/test-production-build.yml

```diff
@@ -8,14 +8,17 @@ on:
 
 jobs:
   Test-Production-Build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     strategy:
       matrix:
-        php-version: [7.3, 7.4, 8.0]
+        php-version: ["7.3", "7.4", "8.0"]
```

Comment:
Without this actions was treating these as numbers and dropping the `.0` from `8.0`.

Reply:
:woman_facepalming: